XamarinCommunityToolkit/Views/SideMenuView/SideMenuView.shared.cs

```diff
@@ -191,6 +191,7 @@ protected override void OnControlInitialized(AbsoluteLayout control)
 			overlayView = SetupMainViewLayout(new BoxView
 			{
 				InputTransparent = true,
+				BackgroundColor = Color.Transparent,
```

Comment:
Just wonder if setting Color instead of BackgroundColor helps as well? if yes, I'd prefer to go ahead with "Color"

Reply:
Good question. Did not try it yet. Will check it and get back when I have an answer

Reply:
So, you're guess was right! Setting the color also solves the problems and is not affecting the other platforms. Pushed the change just now